tests/test_models/mock_llm/config.pbtxt

```diff
@@ -29,14 +29,19 @@ backend: "python"
 
 input [
   {
-    name: "TEXT_INPUT"
+    name: "text_input"
```

Comment:
The input names are being made lowercase to match what GenAi-Perf generates for TRT-LLM/vLLM. Max_tokens is being added as an arg so that this doesn't break in the future. There is an [upcoming PR](https://github.com/triton-inference-server/client/pull/612) that will make sure that PA returns an error if a user-supplied input does not exist in the model (current behavior is to ignore it).